root-sync/base/crossplane/sample-project-claim.yaml

```diff
@@ -1,7 +1,9 @@
 apiVersion: data-science-portal.phac-aspc.gc.ca/v1alpha1
-kind: Project
+kind: ProjectClaim
 metadata:
-  name: crossplane-project-name
+  name: phx-test-project-claim
 spec:
-  name: crossplane-project-name
-  folder: "folder-id"
+  rootFolderId: "108494461414"
+  folderName: phx-my-folder
+  projectName: phx-my-project
+  projectId: phx-01F7DKCVCVDZN1Z5Q4FWANHHCC
```

Comment:
**Non-blocking**:

This looks longer than what Keith is using. Let's focus on getting the budget ready though. I've created https://github.com/PHACDataHub/sci-portal/issues/111 to track it.